modules/core/src/index.ts

```diff
@@ -9,6 +9,7 @@ export {writeFile, writeFileSync} from './lib/fetch/write-file';
 export {setLoaderOptions} from './lib/api/set-loader-options';
 export {registerLoaders} from './lib/api/register-loaders';
 export {selectLoader, selectLoaderSync} from './lib/api/select-loader';
+export {setGlobalOptions, getGlobalOptions} from './lib/loader-utils/option-utils';
```

Comment:
Exporting new functions from the core API is kind of a big decision and should be "audited":

- Such functions should have docs and tests...
- There is already a documented public [`setLoaderOptions`](https://loaders.gl/modules/core/docs/api-reference/set-loader-options) does that not work?
- The `getGlobalOptions` seems like a hack, only needed, until we have updated `encode()` functions to handle worker writers? (since the `encode()` function is inside core it could call `getGlobalOptions()` without exporting it... If so maybe we can export `_getLoaderOptions()` with an underscore?
- Should it be called `_getLoaderOptions()` to match the public `setLoaderOptions()`?
-